src/libraries/System.Text.Json/tests/Serialization/EnumConverterTests.cs

```diff
@@ -232,6 +233,23 @@ public static void MoreThan64EnumValuesToSerialize()
             }
         }
 
+        [Fact]
+        public static void MoreThan64EnumValuesToSerializeWithNamingPolicy()
+        {
+            var options = new JsonSerializerOptions
+            {
+                Converters = { new JsonStringEnumConverter(new ToLower()) }
+            };
+
+            for (int i = 0; i < 128; i++)
+            {
+                MyEnum value = (MyEnum)i;
+                string asStr = value.ToString().ToLowerInvariant();
+                string expected = char.IsLetter(asStr[0]) ? $@"""{asStr}""" : asStr;
+                Assert.Equal(expected, JsonSerializer.Serialize(value, options));
+            }
+        }
```

Comment:
Does this test covers `EnumConverter<T>.WriteWithQuotes`? I was expecting to see a `Dictionary<MyEnum, int>` since I think that's the scenario that uses the `WriteWithQuotes` method.

Here's the test that I had in mind:
```cs
[Fact]
public static void MoreThan64EnumDictionaryKeysToSerialize()
{
    for (int i = 0; i < 128; i++)
    {
        MyEnum myEnum = (MyEnum)i;
        var dictionary = new Dictionary<MyEnum, int> { { myEnum, i } };
        string expected = $@"{{""{myEnum}"":{i}}}";
        Assert.Equal(expected, JsonSerializer.Serialize(dictionary));
    }
}
```

Reply:
This test actually covers `EnumConverter<T>.Write`.  The false condition of this code wasn't being visited by any tests (although given that we're running different test commands, maybe there's something I'm doing wrong here 😉 )
https://github.com/dotnet/runtime/blob/4bc91cb3805bc4a119fc18f4a32d5126ac72db60/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Value/EnumConverter.cs#L193-L196

Sorry for the confusion.  While I was looking at the WriteWithQuotes branch you were talking about, I came across this other piece of uncovered code.